Add ppsspp for PSP support

sharkwouter opened the pull request:
This adds a patched version of the libretro-ppsspp package found in the Arch repo. The difference is that this package includes the assets and has been patched to use them. It is required for https://github.com/gamer-os/steam-buddy/pull/101

alkazar commented:
Blocked on merging PSP support into Steam Buddy.

sharkwouter commented:
Actually, it would be better to add this as dependency for steam-buddy. Somehow we have a lot of emulators in this file here, even though steam-buddy installs them already.

alkazar commented:
Good point. Not sure if I want to make `libretro-ppsspp-gameros` a dependency of `steam-buddy` though.

sharkwouter commented:
I changed the PR to install the package before steam-buddy and changed the package itself to provide libretro-ppsspp. That means you can add libretro-ppsspp as a dependency for steam-buddy and this package will statisfy that.

I'm not sure what is up with the build error, though. I think that is unrelated.

sharkwouter commented:
It seems the error is indeed completely unrelated to the changes in this PR. Game mode was added to the official repo now. Should we fix this in a different PR?

sharkwouter commented:
I've changed the archive version used to make sure the image can be build with gamemode from the repository, since it's no longer in the chaotic-aur.
